CONTRIBUTING.md

```diff
@@ -1,55 +1,164 @@
 # Contributing
 
-The dance tool and tests run on the host; macOS, Linux, and Windows are expected to work.
-Databases under test (FerretDB and MongoDB) may be running on the host or inside Docker; Docker Compose configuration is provided for convenience but not required.
-In particular, the FerretDB development cycle (fix-compile-run-dance) is faster with it running on the host as it does not involve Docker image building or PostgreSQL restarts.
-Running FerretDB on the host is recommended for that reason.
+This document describes how to run and develop dance tools and tests.
+
+## Types of tests
+
+You can find tests in the `tests` directory.
+We support tests of different types:
+
+* `mongo-go-driver` - to test compatibility with the MongoDB Go driver;
+* `diff` - to demonstrate FerretDB-specific differences;
+* `mongo-tools` - to test compatibility with the tools (like dump and restore).
 
-## Cloning repository
+## How to roll out dance locally
```

Comment:
Roll out?

Reply:
Ok, let's call it "clone", but let's keep "How to"